lib/src/rules/length.ts

```diff
@@ -0,0 +1,29 @@
+export interface LengthParams {
+  length: number;
+}
+
+export function parseLengthParams(customParams: LengthParams, errorMessage: string) {
+  const length = customParams.length === null ? NaN : Number(customParams.length);
+  if (isNaN(length)) {
+    throw new Error(errorMessage);
+  }
+
+  return length;
+}
+
+interface LengthValidatorFunction {
```

Comment:
If you look for `LengthParams` you will see it is actually used by `maxLength` and `minLength` to type `customParams`:

```ts
export function maxLength(value: string, vm, customParams: LengthParams = DEFAULT_PARAMS) {
```

It is also used in specs.